lib/features/popup-menu/CreateMenuProvider.js

```diff
@@ -0,0 +1,111 @@
+import { CREATE_OPTIONS } from './util/OptionsUtil';
+
+/**
+ * This module is a create menu provider for the popup menu.
+ */
+export default function CreateMenuProvider(
+    elementFactory,
+    popupMenu,
+    create,
+    autoPlace,
+    mouse
+) {
+  this._elementFactory = elementFactory;
+  this._popupMenu = popupMenu;
+  this._create = create;
+  this._autoPlace = autoPlace;
+  this._create = create;
+  this._mouse = mouse;
+
+  this.register();
+}
+
+CreateMenuProvider.$inject = [
+  'elementFactory',
+  'popupMenu',
+  'create',
+  'autoPlace',
+  'mouse'
+];
+
+/**
+ * Register create menu provider in the popup menu
+ */
+CreateMenuProvider.prototype.register = function() {
+  this._popupMenu.registerProvider('bpmn-create', this);
+};
+
+/**
+ * Get all entries
+ *
+ * @param {djs.model.Base} element
+ *
+ * @return {Array<Object>} a list of menu entry items
+ */
+CreateMenuProvider.prototype.getPopupMenuEntries = function() {
+
+  const entries = {};
+
+  // map options to menu entries
+  CREATE_OPTIONS.forEach(option => {
+    const {
+      actionName,
+      className,
+      label,
+      target,
+      description,
+      group,
+      search
+    } = option;
+
+    const targetAction = this._createEntryAction(target);
+
+    entries[`create-${actionName}`] = {
+      label,
```

Comment:
We should ensure we translate this label + the group label (cf. https://github.com/bpmn-io/bpmn-js/pull/1810). As of today you translate within the provider, not inside the menu.